ls1intum/Artemis
# `General`: Improve user interface of scores page

### Checklist
#### General


#### Client

### Motivation and Context


Fix some small things related to the ui of the scores table.

### Description

Shorten the headlines of student/team names (to just "Names"), shorten the headline of the indicator for the practice mode (to just "practice").
Fix a clipping error when exporting submissions/repos for smaller screens (see screenshot). Now the first popover closes when the second larger for the actual export is opened.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Programming Exercise

1. Log in to Artemis
2. Navigate to the scores table of the exercise
3. Verify these small fixes

### Review Progress






#### Code Review
#### Manual Tests

### Screenshots


Wrong clipping (the export options should be in the back or even closed again):
<img width="1093" alt="image" src="https://user-images.githubusercontent.com/38322605/227243669-973f8410-7c9f-42ab-a569-3e8bc18f5142.png">
New table headings:
<img width="1728" alt="Bildschirm­foto 2023-03-25 um 09 08 48" src="https://user-images.githubusercontent.com/38322605/227705521-4d325104-e45a-4e61-9673-ab822b625418.png">
